src/seedsigner/controller.py

```diff
@@ -305,11 +305,11 @@ def run(self):
                     self.psbt_parser = None
                     self.psbt_seed = None
                 
-                print(f"back_stack: {self.back_stack}")
+                logger.info(f"back_stack: {self.back_stack}")
```

Comment:
(fussy nit) As you mentioned, we can do a more specific overhaul later, BUT... this is an important one that would kind of bug me if it didn't have a "`\n`" added:
```python
logger.info(f"\nback_stack: {self.back_stack}")
```

```
2024-07-08 18:47:01,418  INFO [seedsigner.controller start (312)]: Executing SeedWordsView({'seed_num': None, 'page_index': 2, 'bip85_data': None})
2024-07-08 18:47:01,812  INFO [seedsigner.controller start (358)]: Appending next destination: SeedWordsBackupTestPromptView({'seed_num': None, 'bip85_data': None})
2024-07-08 18:47:01,816  INFO [seedsigner.controller start (363)]: ------------------------------
2024-07-08 18:47:01,820  INFO [seedsigner.controller start (308)]: 
back_stack: [
     3: SeedWordsBackupTestPromptView({'seed_num': None, 'bip85_data': None})
     2: SeedWordsView({'seed_num': None, 'page_index': 2, 'bip85_data': None})
     1: SeedWordsView({'seed_num': None, 'page_index': 1, 'bip85_data': None})
     0: SeedWordsView({'seed_num': None, 'page_index': 0, 'bip85_data': None})
]
2024-07-08 18:47:01,827  INFO [seedsigner.controller start (312)]: Executing SeedWordsBackupTestPromptView({'seed_num': None, 'bip85_data': None})
2024-07-08 18:47:02,984  INFO [seedsigner.controller start (358)]: Appending next destination: SeedFinalizeView()
2024-07-08 18:47:02,988  INFO [seedsigner.controller start (363)]: ------------------------------
2024-07-08 18:47:02,993  INFO [seedsigner.controller start (308)]: 
back_stack: [
     4: SeedFinalizeView()
     3: SeedWordsBackupTestPromptView({'seed_num': None, 'bip85_data': None})
     2: SeedWordsView({'seed_num': None, 'page_index': 2, 'bip85_data': None})
     1: SeedWordsView({'seed_num': None, 'page_index': 1, 'bip85_data': None})
     0: SeedWordsView({'seed_num': None, 'page_index': 0, 'bip85_data': None})
]
```

Reply:
done in https://github.com/SeedSigner/seedsigner/pull/558/commits/19c4d1f4bdaecf292472187adc048c98ed57abb4